include/irqmgr.h

```diff
@@ -3,6 +3,17 @@
 
 #include "ultra64.h"
 
+#define OS_SC_RETRACE_MSG       1
+#define OS_SC_DONE_MSG          2
+#define OS_SC_NMI_MSG           3 // name is made up, 3 is OS_SC_RDP_DONE_MSG in the original sched.c
+#define OS_SC_PRE_NMI_MSG       4
+
+typedef enum {
+    IRQ_RESET_IDLE,
+    IRQ_RESET_PRENMI,
+    IRQ_RESET_NMI
+} IrqResetStatus;
```

Comment:
it's a bit more verbose but imo the prefix should be the full `IRQ_RESET_STATUS_` for clarity, and consistency with overall enum definitions